pennylane_qiskit/vqe_runtime_runner.py

```diff
@@ -283,39 +163,42 @@ def vqe_runner(
             )
             x0 = 2 * np.pi * np.random.rand(num_params)
 
-        inputs["x0"] = x0
+        inputs["ansatz"] = ansatz_circ
+        inputs["initial_parameters"] = x0
 
     # Transform the PennyLane hamilonian to a suitable form
     hamiltonian = hamiltonian_to_list_string(hamiltonian, wires)
 
-    inputs["hamiltonian"] = hamiltonian
+    inputs["operator"] = PauliSumOp.from_list(hamiltonian)
+
+    # Set the optimizer
+    if optimizer == "SPSA":
+        inputs["optimizer"] = SPSA(**optimizer_config)
+    elif optimizer == "QNSPSA":
+        raise ValueError("QNSPSA is not available for vqe_runner")
```

Comment:
Do you know how it is usually specified in a VQE program?

Reply:
both with and without using the runtime, you would pass an initialized Qiskit optimizer to [VQE](https://qiskit.org/documentation/stubs/qiskit.algorithms.VQE.html), only with this `service.run` wrapper when using their runtime. The problem is that the runtime service does not know how to serialize the fidelity function (generated with qiskit code, mind you!), so I'm failing fast here. As commented, I will follow up to see if they can add support on their end